webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+var fs = require('fs');
+var copyright = fs.readFileSync('lib/copyright.frag', 'utf8');
+
+module.exports = {
+    entry: './lib/index.js',
+    output: {
+        path: path.join(__dirname, 'dist', 'browser'),
+        filename: 'OktaAuth.min.js',
```

Comment:
I kept it as OktaAuth.min.js for now to keep the changes to a minimum for now. The OktaAuth.min.js naming convention matches our old output.